docs/src/define-routes/https.md

```diff
@@ -233,43 +233,43 @@ depending on the region.
 Since this pattern for generated URLs should remain similar even if it may change slightly,
 your branch names should be no more than 20 characters.
 
-### DNS Challenge
+### DNS challenge
 
-To provide a valid SSL-certificate,
-Let's Encrypt needs to make sure that the requester is entitled to receive the SSL-certificate it asked for
-(usually through the presence of a specific token on the DNS zone of that domain).
-
-This ownership verification is achieved through the so called _Challenge_ step,
-more background information can be found in the [Let's Encrypt Documentation](https://letsencrypt.org/docs/challenge-types/).
+To provide a valid SSL certificate,
+Let's Encrypt checks that the requester is entitled to receive the SSL certificate it asked for.
+It verifies ownership through a _challenge_.
+For Platform.sh projects, it involves a [HTTP-01 challenge](https://letsencrypt.org/docs/challenge-types/#http-01-challenge).
 
 If you include them in your [routes definition](./_index.md),
 Platform.sh checks that both the `example.platform.sh` and `www.example.platform.sh` domains are pointing to your project.
```

Comment:
:hammer_and_wrench:
Why not use the [recommended example](https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#use-clear-examples)?
```suggestion
Platform.sh checks that both the `example.com ` and `www. example.com ` domains are pointing to your project.
```

Reply:
The reason I didn't was because this wasn't a comprehensive review of what was there, just the relevant bits. I didn't edit these lines, for example.